pydantic_xml/serializers/factories/model.py

```diff
@@ -283,7 +283,12 @@ def from_core_schema(cls, schema: pcs.ModelSchema, ctx: Serializer.Context) -> '
         assert issubclass(model_cls, pxml.BaseXmlModel), "unexpected model type"
 
         name = ctx.entity_path or model_cls.__xml_tag__ or ctx.field_alias or ctx.field_name or model_cls.__name__
-        ns = ctx.entity_ns or model_cls.__xml_ns__ or ctx.parent_ns
+        if ctx.entity_ns is not None:
```

Comment:
There is the similar misbehavior in [mapping](https://github.com/dapper91/pydantic-xml/blob/dev/pydantic_xml/serializers/factories/mapping.py#L69), [wrapper](https://github.com/dapper91/pydantic-xml/blob/dev/pydantic_xml/serializers/factories/wrapper.py#L15), [raw](https://github.com/dapper91/pydantic-xml/blob/dev/pydantic_xml/serializers/factories/raw.py#L16), [primitive#1](https://github.com/dapper91/pydantic-xml/blob/dev/pydantic_xml/serializers/factories/primitive.py#L70), [primitive#2](https://github.com/dapper91/pydantic-xml/blob/dev/pydantic_xml/serializers/factories/primitive.py#L116)

I think it worth to fix them all together. 
This logic is better to be extracted in a function and reused in all the cases like this, for example:

```python
def select_ns(*nss: Optional[str]) -> Optional[str]:
    for ns in nss:
        if ns is not None:
            return ns

    return None
```

Reply:
Thanks for letting me know. I searched the repo for `ctx.entity_ns` and have changed all of them to use `select_ns`